cypress/integration/form-validation/in-list.spec.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
+
+describe('Form validation', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:4200/forms-and-validation/in-a-list');
+    });
+
+    it('should display page title', () => {
+        cy.get('[data-cy=pxb-toolbar]').should('contain', 'In a List');
+    });
+
+    it('should display ip address left aligned', () => {
+        cy.get('[data-cy=ip-address]').invoke('css', 'text-align').should('equal', 'left')
+    });
+
+    it('should display disabled toggle in #424e54 opacity 0.3', () => {
+        cy.get('.mat-slide-toggle-bar').invoke('css', 'background-color').should('equal', 'rgba(66, 78, 84, 0.3)')
+    });
+
+    it('should display enabled toggle in #0088f2 opacity 0.5 ', () => {
+        cy.get('.mat-slide-toggle-bar').click()
+        .invoke('css', 'background-color').should('equal', 'rgba(0, 136, 242, 0.5)')
```

Comment:
I'm not sure why we're doing theme enforcement checks as part of these unit tests.  Is this to increase the code coverage?  If we ever add a dark theme or make theme changes then these tests may fail in the future.

Reply:
yes just to see how it's handled in cypress and reporting. Next quarter we might use cypress mount component for this stuff once the plug-in is flushed out